src/panel-manager.js

```diff
@@ -171,3 +186,30 @@ genotet.panelManager.closeAllPanels = function() {
     .appendTo(tabContent);
   $('#side-panel').css('display', 'none');
 };
+
+/**
+ * Adjust tab height.
+ * @this
+ */
+genotet.panelManager.adjustTabHeight = function() {
+  var tabSelector = $('.sideways');
+  var tabCount = tabSelector.find('li').length - 1;
+  var tabHeight = tabSelector.find('li:nth-child(2) a').outerWidth();
+  var tabContentHeight = $('.tab-content').outerHeight();
+  var newTabHeight = Math.floor((tabContentHeight -
+    this.TAB_HEIGHT_DIFFERENCE - this.TOGGLE_BTN_HEIGHT) / tabCount - 1
```

Comment:
Because there exists an initial invisible panel which is used to be cloned when adding new panels. We should exclude it when calculate new tab height.